instrumentation/rocketmq-client-4.8/library/src/test/groovy/io/opentelemetry/instrumentation/rocketmq/RocketMqClientTest.groovy

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.rocketmq
+
+import base.BaseConf
+import io.opentelemetry.instrumentation.test.InstrumentationSpecification
+import io.opentelemetry.instrumentation.test.LibraryTestTrait
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes
+import org.apache.rocketmq.client.consumer.DefaultMQPushConsumer
+import org.apache.rocketmq.client.producer.DefaultMQProducer
+import org.apache.rocketmq.common.message.Message
+import org.apache.rocketmq.remoting.common.RemotingHelper
+import org.apache.rocketmq.test.listener.rmq.order.RMQOrderListener
+import spock.lang.Shared
+
+import static io.opentelemetry.api.trace.SpanKind.CONSUMER
+import static io.opentelemetry.api.trace.SpanKind.PRODUCER
+import static io.opentelemetry.instrumentation.test.utils.TraceUtils.basicSpan
+import static io.opentelemetry.instrumentation.test.utils.TraceUtils.runUnderTrace
+
+class RocketMqClientTest extends InstrumentationSpecification implements LibraryTestTrait {
```

Comment:
Can this class extend `AbstractRocketMqClientTest` and reuse the test cases? It'd be really great if we had the same set of test cases for library & javaagent instrumentations.

Reply:
@mateuszrzeszutek If reuse javaagent instrumentations test cases，
when use "gradlew :instrumentation:rocketmq-client-4.8:library:test --tests RocketMqClientTest”to test for library instrumentation，the configuration of library.gradle doesn't work.
tasks.withType(Test) {
  jvmArgs "-Dotel.instrumentation.rocketmq-client.experimental-span-attributes=true"
}
Would you pls how to deal with this situation? 

In the process of javaagent instrumentation test,the same test configuration of instrumentation.gradle works.

Reply:
Does library instrumentation test support the configuration?
tasks.withType(Test) {
jvmArgs ...
}
I didn't find examples in this repo.

Reply:
Hi @addname ,
Fix for this is already on the main branch - if you merge/rebase your library tests should no longer fail on missing config.

Reply:
@mateuszrzeszutek  nice, ran successfully!